src/clj_icat_direct/icat.clj

```diff
@@ -268,8 +275,10 @@
 
    Throws:
      It throws an exception if a validation fails."
-  [& {:keys [user zone folder-path entity-type sort-column sort-direction limit offset info-types transaction? user-group-ids]}]
-  (let [groups-table-query (when user-group-ids (apply str "SELECT column1 AS group_user_id FROM (VALUES " (string/join ", " (map #(str "(" % ")") user-group-ids)) ") v"))
+  [& {:keys [user zone folder-path entity-type sort-column sort-direction limit offset info-types transaction?
+             user-group-ids]
+      :or {transaction? true}}]
```

Comment:
The default value for the `transaction?` keyword parameter is the functional change.